src/v3/src/components/AuthContainer/AuthContainer.tsx

```diff
@@ -15,14 +15,14 @@ import { Box } from '@okta/odyssey-react-mui';
 import classNames from 'classnames';
 import { FunctionComponent, h } from 'preact';
 
-import { useWidgetContext } from '../../contexts';
-import { getLanguageCode } from '../../util';
 import style from './style.css';
 
-const AuthContainer: FunctionComponent = ({ children }) => {
+type AuthContainerProps = { languageCode: string; languageDirection?: 'rtl' };
+const AuthContainer: FunctionComponent<AuthContainerProps> = ({
+  languageDirection, languageCode, children,
```

Comment:
do we want to consider allowing `...rest` props and pass them through directly to `Box`?

Reply:
There are two issues with this approach, if we extract children, `rest` will still contain props like `jsx` and `ref` which are incompatible with the `Box` component. Also, we disallow spreading props via lint so we would also need to disable it for this line if we were to do it. 

![image](https://user-images.githubusercontent.com/97472729/219412921-ca50cfff-77e8-4824-990a-9eed2c5cb1a3.png)